packages/squidex/schema/schemas/calendars.json

```diff
@@ -34,9 +34,53 @@
         "partitioning": "invariant",
         "properties": {
           "fieldType": "String",
+          "allowedValues": [
+            "#B1365F",
+            "#5C1158",
+            "#711616",
+            "#691426",
+            "#BE6D00",
+            "#B1440E",
+            "#853104",
+            "#8C500B",
+            "#754916",
+            "#88880E",
+            "#AB8B00",
+            "#856508",
+            "#28754E",
+            "#1B887A",
+            "#28754E",
+            "#0D7813",
+            "#528800",
+            "#125A12",
+            "#2F6309",
+            "#2F6213",
+            "#0F4B38",
+            "#5F6B02",
+            "#4A716C",
+            "#6E6E41",
+            "#29527A",
+            "#2952A3",
+            "#4E5D6C",
+            "#5A6986",
+            "#182C57",
+            "#060D5E",
+            "#113F47",
+            "#7A367A",
+            "#5229A3",
+            "#865A5A",
+            "#705770",
+            "#23164E",
+            "#5B123B",
+            "#42104A",
+            "#875509",
+            "#8D6F47",
+            "#6B3304",
+            "#333333"
+          ],
           "isUnique": false,
           "inlineEditable": false,
-          "editor": "Color",
+          "editor": "Radio",
```

Comment:
This is gonna be a nice editing experience :joy: 
Can we do a different label and value?
If so, could use color 'names' at least: https://webmastermind.net/knowledgebase.php?action=displayarticle&id=15

Reply:
No way to have a different value and label. We can create a mapping on the backend :shrug: